waspc/src/Wasp/Generator/JsImport.hs

```diff
@@ -8,19 +11,66 @@ import qualified StrongPath as SP
 import qualified Wasp.AppSpec.ExtImport as AS.ExtImport
 import Wasp.Generator.ExternalCodeGenerator.Common (GeneratedExternalCodeDir)
 
-getJsImportDetailsForExtFnImport ::
-  -- | Path to generated external code directory, relative to the directory in which file doing the importing is.
-  Path Posix (Rel a) (Dir GeneratedExternalCodeDir) ->
-  AS.ExtImport.ExtImport ->
+data ImportStatement = ImportStatement
+  { _extImportName :: AS.ExtImport.ExtImportName,
+    _path :: FilePath,
+    _importAlias :: Maybe String
```

Comment:
Yep, we had it implicitly because for routes in `router.js` we used page name instead of the actual import name when generating the `import` statements.